src/parser/GeoPoint.cpp

```diff
@@ -95,12 +96,19 @@ GeoPoint GeoPoint::fromBitRepresentation(T bits) {
   return {lat, lng};
 };
 
+// _____________________________________________________________________________
 std::string GeoPoint::toStringRepresentation() const {
   // Extra conversion using std::to_string to get more decimals
   return absl::StrCat("POINT(", std::to_string(getLng()), " ",
                       std::to_string(getLat()), ")");
 };
 
+// _____________________________________________________________________________
 std::pair<std::string, const char*> GeoPoint::toStringAndType() const {
   return std::pair(toStringRepresentation(), GEO_WKT_LITERAL);
 };
+
+// _____________________________________________________________________________
+bool GeoPoint::operator==(const GeoPoint& other) const {
```

Comment:
As these are the only members, you can default this operator, but the following has to be in the header:

`bool operator==(const GeoPoint& other) const = default`
(Very handy C++20 feature).